test/input/common/akka.conf

```diff
@@ -11,12 +11,6 @@ beam-agent-scheduler-pinned-dispatcher {
   thread-pool-executor.allow-core-timeout=off
 }
 
-zonal-parking-manager-pinned-dispatcher {
```

Comment:
@REASY what do you think?

Reply:
I'm going to restore this zonal...dispatcher for the configured ZonalParkingManager (if the user defines to use the Default PM)
What about the new Hierarchical ParkingManager. I think it's better to assign a common Akka dispatcher to it in order to reduce Threading overhead. Though we may want to assign a separate dispatcher for debugging purposes.

Reply:
Returned back zonal dispatcher for Zonal PM